elements/pfelement/src/pfelement.js

```diff
@@ -230,10 +230,10 @@ class PFElement extends HTMLElement {
   }
 
   /**
-   * Returns an array with all the slot with the provided name defined in the light DOM.
-   * If no value is provided (i.e., `this.getSlot()`), it returns all unassigned slots.
+   * Given a slot name, returns elements assigned to the slot.
+   * If no value is provided (i.e., `this.getSlot()`), it returns all children without a slot attribute.
```

Comment:
```suggestion
   * If no value is provided (i.e., `this.getSlot()`), it returns all children not assigned to a slot.
```

The assignment is the correct terminology when referring to slots.  Suggest adding that back in.

Reply:
I was thinking that may not be easy to understand for folks new to web components, how do you feel about having a parenthetical?
```
* If no value is provided (i.e., `this.getSlot()`), it returns all children not assigned to a slot (without a slot attribute).
```